site/content/docs/main/csi-snapshot-data-movement.md

```diff
@@ -241,14 +241,11 @@ kubectl -n velero get datadownloads -l velero.io/restore-name=YOUR_RESTORE_NAME
 
 ## Limitations
 
-- CSI and CSI snapshot support both file system volume mode and block volume mode. At present, Velero built-in data mover doesn't support 
-block mode volume or volume snapshot. 
+- CSI and CSI snapshot support both file system volume mode and block volume mode. At present, block mode is only supported for non-Windows platforms, because the block mode code invokes some system calls that are not present in the Windows platform.  
```

Comment:
I think it is better to have inclusive language like we only support linux platform, because non-Windows could imply macOS block mode support.

Reply:
This block is added by #7013, here in this PR, I just moved it since the first clarification and this one are contradictory.
Loop in @blackpiglet who had created #7013.

Reply:
It's a little tricky to describe the compatibility of the block mode support. `stat_t`